tests/pr_test.go

```diff
@@ -8,24 +8,29 @@ import (
 	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
 )
 
-// Use existing resource group
-const resourceGroup = "geretain-test-resources"
 const defaultExampleTerraformDir = "examples/default"
 
+// Use existing group for tests
+const resourceGroup = "geretain-test-event-streams"
+
+// Not all regions provide cross region support so value must be hardcoded https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-service-availability.
+const region = "us-south"
```

Comment:
No need to lock into us-south (yet). All the regions in https://github.com/terraform-ibm-modules/common-dev-assets/blob/main/common-go-assets/icd-region-prefs.yaml support BYOK, so can you set it up to pick a region from here (check out the internal [elasticsearch module](https://github.ibm.com/GoldenEye/icd-elasticsearch-module/blob/master/tests/pr_test.go#L21) for example)